From 064f24827209bbe93d13ca0134de484c7c9c4f86 Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Mon, 4 Jan 2016 12:19:38 +0200
Subject: [PATCH 0825/1383] fsl-dpaa2: mac: Add counter for "good" egress
 frames

Now available with the 0.8.1 Flibs.

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl-dpaa2/mac/mac.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index 826c499..8e7d4ee 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -273,6 +273,7 @@ static struct {
 	{DPMAC_CNT_ING_OVERSIZED,		"rx oversized"},
 	{DPMAC_CNT_ING_VALID_PAUSE_FRAME,	"rx pause"},
 	{DPMAC_CNT_ING_BYTE,			"rx bytes"},
+	{DPMAC_CNT_ENG_GOOD_FRAME,		"tx frames ok"},
 	{DPMAC_CNT_EGR_UCAST_FRAME,		"tx u-cast"},
 	{DPMAC_CNT_EGR_MCAST_FRAME,		"tx m-cast"},
 	{DPMAC_CNT_EGR_BCAST_FRAME,		"tx b-cast"},
-- 
2.8.1

